data/pokedex.ts

```diff
@@ -40,8 +40,9 @@ export const BattlePokedex: {[speciesid: string]: SpeciesData} = {
 		prevo: "Ivysaur",
 		evoLevel: 32,
 		eggGroups: ["Monster", "Grass"],
-		otherFormes: ["Venusaur-Mega", "Venusaur-Gmax"],
+		otherFormes: ["Venusaur-Mega"],
 		formeOrder: ["Venusaur", "Venusaur-Mega", "Venusaur-Gmax"],
```

Comment:
tbh I think it makes more sense to remove Gmaxes from `formeOrder` than from `otherFormes`.

Reply:
Ideally both. Maybe mark Gmax formes with `isNonstandard: "Gigantamax"`?

Reply:
They are literally not formes in any sense of the word, I'd rather not remove them from formeOrder because I believe the sprite project needs them (@scheibo can you confirm/deny?). The only reason they have entries in `pokedex.ts` and `formats-data.ts` is for backwards compatibility and convenience for the teambuilder and random battles.

Reply:
I will add a `isNonstandard: "Gigantamax"` option though.

Reply:
I thought they were only added into `formeOrder` because Zarel had wanted them there (https://github.com/smogon/pokemon-showdown/pull/6669#issuecomment-626109797). Not entirely sure though, the discussion might be in #art? Shinies are not formes either and the sprite project handles them. I dont think non-formes belong in `formeOrder`.

Reply:
> I will add a isNonstandard: "Gigantamax" option though.

wait why. I specifically removed this. GIGANTAMAXES ARE NOT NONSTANDARD. please do not mark them as such

Reply:
I wanted them in `otherFormes`. My comment you linked to was specifically about a strategy to use `otherFormes` rather than having a separate `formeOrder` array. If we do have a separate `formeOrder` array, it makes sense to leave off non-formes like Gigantamaxes from it.

Reply:
Also, with this PR comes a new strategy for handling gigantamaxes that I no longer think it's necessary to have gigantamaxes in `otherFormes`. Well, you could make arguments either way, it's just not a slam-dunk anymore.

Reply:
I'm not opposed to just using `isGigantamax` as a stopgap without adding the `isNonstandard` flag.